UseLatexMk.cmake

```diff
@@ -131,7 +132,6 @@ function(add_latex_document)
   set(OPTION REQUIRED EXCLUDE_FROM_ALL BUILD_ON_INSTALL)
   set(SINGLE SOURCE TARGET INSTALL)
   set(MULTI FATHER_TARGET RCFILE)
-  include(CMakeParseArguments)
```

Comment:
Is this implicitly included now? I thought this is still required. It is for sure unnecessary in the Dune context, where it has been included 100x times before we hit this line, but it might be missing in other contexts.

Reply:
Since 3.5 this is a native command.

> New in version 3.5: This command is implemented
> natively. Previously, it has been defined in the
> module [CMakeParseArguments](https://cmake.org/cmake/help/latest/module/CMakeParseArguments.html#module:CMakeParseArguments).`

Source: https://cmake.org/cmake/help/latest/command/cmake_parse_arguments.html

Reply:
Ah, that is good news. I was looking at the module doc, which did not have this bit...